src/modules/navigator/rtl.cpp

```diff
@@ -300,11 +300,6 @@ RTL::set_rtl_item()
 
 	reset_mission_item_reached();
 
-	/* execute command if set */
-	if (!item_contains_position(&_mission_item)) {
-		issue_command(&_mission_item);
-	}
-
```

Comment:
@LorenzMeier Removing this breaks the backtransition before landing in rtl on a vtol